### PR TITLE
feature/1403-homepage-reflow

### PR DESCRIPTION
**Pull request by jonhindle**
### Context

> Does this issue have a Trello card?
https://trello.com/c/r9grt0oc/1403-new-homepage-reflowing

> If this is an issue, do we have steps to reproduce?
When the site is resized the home elements become distorted and difficult to use.

### Intent

> What changes are introduced by this PR that correspond to the above card?
Attempted to reflow the items as shown in the relevant figma

> Would this PR benefit from screenshots?
Screenshot 2022-08-16 at 09 21 35
Screenshot 2022-08-16 at 09 21 07

### Considerations

> Is there any additional information that would help when reviewing this PR?
Currently the page reflows under 1200px.

> Are there any steps required when merging/deploying this PR?
n/a

### Checklist

- [ ] This PR contains **only** changes related to the above card
- [ ] Tests have been added/updated to cover the change
- [ ] Documentation has been updated where appropriate
- [ ] Tested in Development

### CODE REVIEW

File: assets/sass/components/_key-info-items.scss [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  margin-left: govuk-spacing(2);
+  flex-direction: column;
+  @include govuk-media-query($until: $width-override) {
+    flex-direction: row;
+    margin-left: 0;
+
+    > .govuk-hub-key-info-item {
+      padding: 0 govuk-spacing(1);
+      width: 25%;
+      width: 25%;
```
Comment (jonhindle) [inline assets/sass/components/_key-info-items.scss:12]: duplication 🙈

File: assets/sass/components/_home-content.scss [hunk 2]
```diff
@@ -282,22 +282,47 @@
...
+
+.govuk-hub-home-updates-feature,
+.govuk-hub-home-updates-content {
+  max-width: 50%;
+  flex-grow: 3;
+  @include govuk-media-query($from: $width-override) {
+    max-width: 450px;
+  }
+  @include govuk-media-query($until: tablet) {
+    max-width: 0;
```
Comment (BarryGeeMOJ) [inline assets/sass/components/_home-content.scss:311]: how does this work? max-width of 0 but a min-width of 100%

File: server/views/pages/home-new.html [hunk 3]
```diff
@@ -32,24 +32,20 @@
...
-          <div class="govuk-grid-column-one-half govuk-!-padding-0 govuk-!-padding-right-3">
-            <!-- 450px column, small update links -->
-            {% if updatesContent | length %}
-              {{ updateItems(updatesContent, updatesContentHideViewAll) }}
-            {% endif %}
-          </div>
-        </div>
+    <div class="govuk-grid-row govuk-!-margin-0 govuk-body govuk-hub-home-updates">
+      <div class="govuk-hub-home-updates-feature">
+        <!-- 450px column, large update link -->
```
Comment (BarryGeeMOJ) [inline server/views/pages/home-new.html:37]: do we need this comment?

File: server/views/pages/home-new.html [hunk 4]
```diff
@@ -32,24 +32,20 @@
...
+        {% if largeUpdateTile %}
+          {{ contentTileFeatured(largeUpdateTile) }}
+        {% endif %}
+      </div>
+      <div class="govuk-hub-home-updates-content">
+        <!-- 450px column, small update links -->
```
Comment (BarryGeeMOJ) [inline server/views/pages/home-new.html:43]: do we need this comment?

**Review by BarryGeeMOJ — CHANGES_REQUESTED** [review]

**Review by BarryGeeMOJ — APPROVED** [review]

**Pull request merged**